docs/pages/docs/providers/rust.md

```diff
@@ -46,3 +46,9 @@ These directories are cached between builds
 - Build: `~/.cargo/git`
 - Build: `~/.cargo/registry`
 - Build: `target`
+
+## [Workspaces](https://doc.rust-lang.org/book/ch14-03-cargo-workspaces.html)
+
+Nixpacks will auto-detect if you are using Cargo Workspaces.
```

Comment:
This should probably state that it respects `default_members`.

Reply:
Done in https://github.com/railwayapp/nixpacks/pull/402/commits/d853c9a1af7d579e217df2363b481395cc70b193